.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets" : ["es2015", "react"]
```

Comment:
Consider using the env preset here because es2015 has been deprecated.

Reply:
Thanks for this tip. Fixed!